Updates metadata. Makes check if station is in metadata

dulte opened the pull request:
Small changes to eeareader

heikoklein commented on src/pyaro_readers/eeareader/EEATimeseriesReader.py:
```diff
@@ -73,6 +78,7 @@ def __init__(
         self._set_filters(filters)
 
         self.metadata = self._read_metadata(filename)
+        self.datafile = self._read_data()
```
both `datafile` and `_read_data` and `data.toml` are very confusing here. It is everything but `data`=measurement of e.g. 17.35µg/m³, please find a better name. `cfg` or `setup`? And definitely don't call something for `file` which is neither a filename nor filehandle.

dulte commented:
I've added the possibility to change units with a factor. Factor is added to data.toml. Might want a separate file for these factors

dulte commented:
I'm not sure with the files fail due to not finding data.toml. Will have a look next week

heikoklein reviewed (changes requested):
Tests not working (still draft)? Very strange choice of variable/file-names.

dulte commented:
@heikoklein . Yes, right before you answered I changed it to a draft. The tests fail because they can't find data.toml, and I'm not entirely sure why... going to look at it over the weekend.

I've added a simple unit conversion field as well in data.toml. It seems to work, but I have to add some tests for that as well before the pr is ready for review. It might also be the wrong way of doing a unit change (changing it without making a new variable). So some work before the pr is ready

dulte commented:
Note to self: remove unit change from data.toml, and add it as a possible argument to the reader instead. 

Or maybe it as a wrapper to pyaro, similar to the variable name change wrapper, so all readers can do unit change.

dulte commented:
@heikoklein. Fixed some problems, and made some new ones. But it is now ready for review

heikoklein reviewed (approved):
Looks good, except for linting. You should really fix your black setup.
